xtokens/src/lib.rs

```diff
@@ -571,6 +574,7 @@ pub mod module {
 					&dest,
 					Some(T::SelfLocation::get()),
 					dest_weight,
+					true,
```

Comment:
See the comment above, I just set `true` here, then this first xcm will always using `teleport`. Do we need use configuration setting like `TrustedTeleporters` to decide whether use teleport or not?